lgsf/__init__.py

```diff
@@ -5,20 +5,20 @@ def setup():
     print("CALLED")
 
 
-LOGO = """          %%%%%%%%%          
-      %%%%%%%%%%%%%%%%%      
-    %%%%%%%%%%%%%%%%%%%%%    
-  %%%%%%%%%%%%%%%%%%%%%%%%%  
- %%%%%%%%%%%%%%%%%%%%   %%%% 
+LOGO = """          %%%%%%%%%
+      %%%%%%%%%%%%%%%%%
+    %%%%%%%%%%%%%%%%%%%%%
```

Comment:
$ python manage.py still prints the logo nicely on my terminal with the whitespace stripped. Can tell black to ignore if it's necessary.